sol-did/programs/sol-did/src/utils.rs

```diff
@@ -1,19 +1,39 @@
 use crate::constants::DID_SOL_PREFIX;
+use crate::{id, DID_ACCOUNT_SEED};
+use anchor_lang::prelude::*;
+use lazy_static::lazy_static;
+use regex::Regex;
 use solana_program::keccak;
 use solana_program::secp256k1_recover::Secp256k1Pubkey;
 
+lazy_static! {
```

Comment:
One concern is this warning on build:

```
cargo-build-bpf child: cargo +bpf build --target bpfel-unknown-unknown --release
Error: Function _ZN5regex4exec11ExecBuilder5build17h83ceff945f02fa86E Stack offset of 7688 exceeded max offset of 4096 by 3592 bytes, please minimize large stack variables
```

Guessing this is caused by this block. Not sure what effect it has on compute . However I'm wondering if we should do this differenty, avoid using a regex by just using indexof/substring or similar string manipulation?

Reply:
hm, funny I used this `lazy_static` also before and ran into the same problem (when trying to return the `owner()` as a static Pubkey. Afterwards I decided against it.

Reply:
Removed after discussion